setup.c

```diff
@@ -42,7 +42,7 @@ void setup(void)
 
 	// enable RNG
 	rcc_periph_clock_enable(RCC_RNG);
-	RNG_CR |= RNG_CR_IE | RNG_CR_RNGEN;
+	RNG_CR |= RNG_CR_RNGEN;
```

Comment:
i went ahead and added the setupApp change (with comment). i have to run right now, but will research further and test later today. i think this is ok, though pending something unforeseen.